.github/workflows/integration.yml

```diff
@@ -34,14 +34,44 @@ jobs:
   celeborn_integration_test:
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v2
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          submodules: recursive
+          fetch-depth: 0
       - uses: actions/setup-java@v2
         name: Setup JDK with Maven
         with:
           distribution: zulu
           java-version: 8
           cache: maven
           check-latest: false
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.12"
+          check-latest: true
+      - name: Setup Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.10.0
+      - name: Setup chart-testing
+        uses: ./.github/actions/chart-testing-action
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+          if [[ -n "$changed" ]]; then
+            echo "changed=true" >> "$GITHUB_OUTPUT"
+          fi
+      - name: Run chart-testing (lint)
+        if: steps.list-changed.outputs.changed == 'true'
+        run: ct lint --target-branch ${{ github.event.repository.default_branch }} --check-version-increment=false --validate-maintainers=false
+      - name: Run unit tests
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm plugin install https://github.com/helm-unittest/helm-unittest.git --version=0.5.1
```

Comment:
let's leave a comment here to mention the version should be synced with docs

Reply:
> let's leave a comment here to mention the version should be synced with docs

Done.